resources/js/data-access/memo-api/groups/System.ts

```diff
@@ -23,6 +23,8 @@ export namespace System {
     ({
       queryFn: ({signal}) => getFacilitiesList({signal}),
       queryKey: keys.facilityList(),
+      // Prevent refetching on every page.
+      refetchOnMount: false,
```

Comment:
dlaczego dla innych już nie chcemy tego jako default?

Reply:
Nie wiem czy wyjaśniłeś dlaczego refetchOnMount: true, bo stale time załatwia kwestię bezczynnej przeglądarki?

Bo tak to się zaczyna robić niespójnie, dla wybranych query nagle wyjątek. A ja np. używam tego query do edycji placówki.

A co to w ogóle znaczy 'mount' w solidzie? Przy tworzeniu każdego komponentu czy jakoś rozróżnia komponenty w Route ? To nie wyłącza cache prawie całkiem?

Reply:
Omówione na discordzie.